test/controllers/samples_bulk_upload_test.rb

```diff
@@ -334,6 +334,57 @@ class SamplesBulkUploadTest < ActionDispatch::IntegrationTest
     assert_equal 0, Sample.where(name: "RR004_water_2_S23A").length
   end
 
+  # If the metadata field isn't supported by the sample's host genome, throw an error.
+  test 'bulk upload invalid key for  host genome' do
+    post user_session_path, params: @user_params
+    post user_session_path, params: @user_params
```

Comment:
why is this line twice? copy paste error?

Reply:
Good catch!